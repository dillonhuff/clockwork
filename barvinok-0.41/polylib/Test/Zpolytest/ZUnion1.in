#readMat["(i,j->2i+1,2i+j)"]
#readMat["(i,j->2i,2i+j)"]
M 2
3 3
2 0 1 
2 1 0
0 0 1
3 3
2 0 0
2 1 0
0 0 1
#readDom["{i,j| 1<= i,j<=10}"]
#readDom["{i,j| 1<= i,j<=10}"]
D 2
4 4
	  1 1 0 -1
          1 0 1 -1
          1 -1 0 10
          1 0 -1 10
 4 4
	  1 1 0 -1
          1 0 1 -1
          1 -1 0 10
          1 0 -1 10
F 8



