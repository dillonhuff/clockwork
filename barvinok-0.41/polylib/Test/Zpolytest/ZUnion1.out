
ZPOLYHEDRON: Dimension 2 

LATTICE: 
3 3
    2    0    1
    0    1    0
    0    0    1
POLYHEDRON Dimension:2
           Constraints:4  Equations:0  Rays:4  Lines:0
Constraints 4 4
Inequality: [    1    0   -1 ]
Inequality: [   -2    1   -1 ]
Inequality: [   -1    0   10 ]
Inequality: [    2   -1   10 ]
Rays 4 4
Vertex: [   10   21 ]/1
Vertex: [    1    3 ]/1
Vertex: [    1   12 ]/1
Vertex: [   10   30 ]/1

UNIONED with

ZPOLYHEDRON: Dimension 2 

LATTICE: 
3 3
    2    0    0
    0    1    0
    0    0    1
POLYHEDRON Dimension:2
           Constraints:4  Equations:0  Rays:4  Lines:0
Constraints 4 4
Inequality: [    1    0   -1 ]
Inequality: [   -2    1   -1 ]
Inequality: [   -1    0   10 ]
Inequality: [    2   -1   10 ]
Rays 4 4
Vertex: [   10   21 ]/1
Vertex: [    1    3 ]/1
Vertex: [    1   12 ]/1
Vertex: [   10   30 ]/1
